File: api/code-samples/python/klay/account/getAccountKey.py
import json

from klay.opensdk_python_klay.configuration import Configuration
from klay.opensdk_python_klay.api_client import ApiClient
from klay.opensdk_python_klay.apis.tags.klay_api import GetAccountKey

host = "https://api.baobab.klaytn.net:8651"

address = "0x3111a0577f322e8fb54f78d9982a26ae7ca0f722"
blockTag = "latest"

klay = GetAccountKey(ApiClient(configuration=Configuration(host=host)))
klay_response = klay.get_account_key(address, blockTag)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/account/getAccountKey.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const address = '0xa36a5fdc679ecaabe057556ccec2f3558068bdc8'
    const blockNumberOrHash = 'latest'

    provider.klay.getAccountKey(address, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });

}
)()